test/boost/s3_test.cc

```diff
@@ -626,6 +626,44 @@ SEASTAR_THREAD_TEST_CASE(test_object_reupload) {
     }
 }
 
+void test_download_data_source(const client_maker_function& client_maker, unsigned chunks) {
+    const sstring name(fmt::format("/{}/testdatasourceobject-{}", tests::getenv_safe("S3_BUCKET_FOR_TEST"), ::getpid()));
+
+    testlog.info("Make client\n");
+    semaphore mem(16<<20);
+    auto cln = client_maker(mem);
+    auto close_client = deferred_close(*cln);
+
+    static constexpr unsigned chunk_size = 1000;
+    testlog.info("Upload object");
+    auto rnd = tests::random::get_bytes(chunk_size);
+    {
+        auto out = output_stream<char>(cln->make_upload_sink(name));
+        auto close = seastar::deferred_close(out);
+
+        for (unsigned ch = 0; ch < chunks; ch++) {
+            out.write(reinterpret_cast<char*>(rnd.begin()), rnd.size()).get();
+        }
+        out.flush().get();
+    }
+
+    testlog.info("Download object");
```

Comment:
```suggestion
    testlog.info("Test: Download object");
```

Reply:
No need, the testlog will prepend `test - ` on its own